### PR TITLE
feat: add simple-pagination

### PR DESCRIPTION
**Pull request by victorhugorch**

### CODE REVIEW

File: src/transactions/pagination-data.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * @param {Object} connection
+ * @param {String} tableName
+ * @param {String} predicate?
+ * @returns {Array||Object} results || result
+ */
+export default async (connection, tableName, predicate = false) => {
+  const { page, pageSize } = predicate
+
+  return rethinkdb.table(tableName)
+    .orderBy('teste')
```
Comment (clucasalcantara) [inline src/transactions/pagination-data.js:29]: 👀  We really need this orderBy here?

File: src/transactions/database.js [hunk 2]
```diff
@@ -29,6 +30,16 @@ export const createDatabase = async (connection, dbName, done) => {
   return false
 }
 
+/**
+ * connection database
+ */
+export const getConnection = async (database = '') =>
+  createLink({
+    host: process.env.ENV === 'mock' ? '172.18.0.2' : 'localhost',
+    port: process.env.ENV === 'mock' ? '28015' : '55001',
```
Comment (clucasalcantara) [inline src/transactions/database.js:39]: I really think that this port should be fixed as 28015 and the fix should be done in the docker container mapping correctly the port exposure

**Review by clucasalcantara — CHANGES_REQUESTED** [review]
AWESOME! 👯 

Just left a few minors to fix :D

File: package.json [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
-  "version": "1.4.2",
+  "version": "1.4.3",
   "description": "a library to make easier the operations handling in rethinkdb",
   "main": "dist/index.js",
   "scripts": {
     "example-bootstrap": "babel-node src/example/bootstrap/",
     "run-example": "babel-node src/example",
     "build": "babel src/ --out-dir dist/ --source-maps --copy-files",
     "test:ci": "ENV=mock ava",
+    "test:local": "ENV=local ava",
```
Comment (clucasalcantara) [inline package.json:11]: When fixing your docker to run at 28015 you don't gonna need this script as well :D

**Comment by victorhugorch** [comment]
I added the suggested changes. Thanks for the review! <3

**Review by clucasalcantara — APPROVED** [review]

**Pull request merged**